lib/puppet/defaults.rb

```diff
@@ -1948,7 +1952,7 @@ def self.default_vendormoduledir
         require 'puppet/node/facts'
         if value
           Puppet::Resource::Catalog.indirection.cache_class = :store_configs
-          Puppet.settings.override_default(:catalog_cache_terminus, :store_configs)
+          settings.override_default(:catalog_cache_terminus, :store_configs)
```

Comment:
@tvpartytonight this was a new change in the merge up, I assume this is correct?

Reply:
this is correct. 💯

Reply:
@tvpartytonight does that count as a +1 for the PR? How would you feel if I merged this?

Reply:
Or, you're also welcome to merge this yourself!!